boards/imix/debug.gdb

```diff
@@ -0,0 +1,6 @@
+
+target remote :3333
+monitor reset
```

Comment:
Depending on the chip, `monitor reset` can sometimes be flaky, and resetting twice usually guarantees that the board is in a fully defined state.